src/fft_convolve.jl

```diff
@@ -44,7 +44,7 @@ function fft_conv!(
     imfilterz!(plan)
 
     (M, N) = size(img)
-    copyto!(output,
+    copyto!(output, # todo: why only 2 of 4 padsize values used here?
```

Comment:
We only need 2 padsize values and `M, N` to determine the outputsize,
one can replace `1:plan.padsize[1]+M` with `end - plan.padsize[2]`.

Reply:
Ok, I made a different simplification.  Thanks!